crates/red_knot_test/src/parser.rs

```diff
@@ -414,7 +416,12 @@ impl<'s> Parser<'s> {
         Ok(())
     }
 
-    fn process_code_block(&mut self, lang: &'s str, code: &'s str) -> anyhow::Result<()> {
+    fn process_code_block(
+        &mut self,
+        lang: &'s str,
+        code: &'s str,
+        backtick_offset: TextSize,
```

Comment:
I changed this to explicitly pass in the offset instead of relying on a specific state of the lexer.